code/mob/living.dm

```diff
@@ -385,8 +385,13 @@
 	if (!QDELETED(W) && (equipped() == W || usingInner))
 		var/pixelable = isturf(target)
 		if (!pixelable)
-			if (istype(target, /atom/movable) && isturf(target.loc))
-				pixelable = TRUE
+			if (istype(target, /atom/movable))
+				if (isturf(target.loc))
+					pixelable = TRUE
+				else if (istype(target.loc, /obj/item/plate))
+					pixelable = TRUE
+				else if (istype(target.loc, /obj/item/reagent_containers/food/snacks/ingredient/pizza_base))
+					pixelable = TRUE
```

Comment:
Well no, again this code should never be getting here! Look into why it's passing range checks instead of adding a snowflake fix here.